spec/sanity/events.js

```diff
@@ -1,38 +1,199 @@
-//
-//        browser.sleep(1000).then(function () { done(); });
-//    });
-//
-//    it('should list events for blueprint', function(done) {
-//        components.ui.events.IndexPage.uncheckAllBlueprints();
-//        components.ui.events.IndexPage.selectBlueprint('nodecellar1');
-//        components.ui.events.IndexPage.selectTimeframe('5 Days');
-//        components.ui.events.IndexPage.show();
-//
-//        browser.sleep(5000).then(function(){ done(); });
-//    });
-//
-//    it('should list events for blueprint and deployment', function(done) {
-//        components.ui.events.IndexPage.uncheckAllBlueprints();
-//        components.ui.events.IndexPage.selectBlueprint('nodecellar1');
-//        components.ui.events.IndexPage.uncheckAllDeployments();
-//        components.ui.events.IndexPage.selectDeployment('deployment1');
-//        components.ui.events.IndexPage.selectTimeframe('5 Days');
-//        components.ui.events.IndexPage.show();
-//
-//        browser.sleep(5000).then(function(){ done(); });
-//    });
-//
-//});
\ No newline at end of file
+'use strict';
+var logger = require('log4js').getLogger('blueprints_spec');
+var components = require('../../src/components/index');
+var events = components.ui.events.page;
+
+describe('logs & events page', function() {
+
+    beforeEach(function(){
+        events.route();
+    });
+    
+    it('should route to events page', function(){
+        expect(browser.getCurrentUrl()).toContain('/logs');
+    });
+
+    //TODO: make it after using api
+    //it('should sort by timestamp desc by default', function(){
+
+    //});
+
+    it('should filter blueprints', function(){
+        //getting number of events on startup
+        var noFiltersEventsCount = events.mainTable.countRows();
+        //Choosing a blueprint with no events
+        events.filters.blueprints.select('nodecellar_undeployed');
+        events.filters.blueprints.getSelected().getText().then(function(text){
+            expect(text).toEqual(['nodecellar_undeployed']);
+            events.filters.blueprints.toggle();
+        });
+        expect(events.mainTable.countRows()).toBe(0);
+
+        //Choosing another blueprint with events
+        events.filters.blueprints.select('nodecellar1');
+        events.filters.blueprints.getSelected().getText().then(function(text){
+            expect(text).toEqual(['nodecellar1', 'nodecellar_undeployed']);
+            events.filters.blueprints.toggle();
+        });
+        expect(events.mainTable.countRows()).toBe(noFiltersEventsCount);
+    });
+
+    it('should filter deployments', function(){
+        //Choosing a deployment with no events
+        events.filters.deployments.select('installed_deployment [nodecellar1]');
+        events.filters.deployments.getSelected().getText().then(function(text){
+            expect(text).toEqual(['installed_deployment [nodecellar1]']);
+            events.filters.deployments.toggle();
```

Comment:
Perhaps we should have `getSelectedText` that will also toggle? like `getSelectedOptions` and then it would be

```
expect(...getSelectedText()).toBe('xyz'); 
```

what do you think?